staging/src/k8s.io/metrics/pkg/client/custom_metrics/scheme/register.go

```diff
@@ -26,12 +26,19 @@ import (
 	cmv1beta2 "k8s.io/metrics/pkg/apis/custom_metrics/v1beta2"
 )
 
+// GroupName is the group name use in this package.
 const GroupName = cmv1beta1.GroupName
 
+// SchemeGroupVersion is group version used to register these objects.
 var SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: runtime.APIVersionInternal}
 
+// Scheme is the runtime.Scheme to which all cmint api types are registered.
```

Comment:
What's the `cmint` mean? Is there a typo?

Reply:
This is a poor abbreviation of “custom metrics api” and has been corrected.